html/pfappserver/lib/pfappserver/Form/Config/Pf.pm

```diff
@@ -141,6 +141,16 @@ sub field_list {
                 $field->{options} = \@options;
                 last;
             };
+            $type eq 'timezone' && do {
+                $field->{type} = 'Select';
+                $field->{element_class} = ['chzn-deselect'];
+                $field->{element_attr} = {'data-placeholder' => 'Select a timezone'};
+                my @timezones = DateTime::TimeZone->all_names();
```

Comment:
Just in case i prefer to add DateTime::TimeZone as a dep for Debian and maybe for centos